README.md

```diff
@@ -65,7 +53,7 @@ Every project runs differently. Follow instructions in the main `README.md` docu
 Do not forget about creating unit and acceptance tests if needed. For the unit tests, follow the instructions specified in the main `README.md` document of the given project. For the details concerning the acceptance tests, go to the corresponding directory inside the `tests` directory.
 Find the information on how to run changes on the cluster without a Docker image in the [Develop a service locally without using Docker](docs/kyma/docs/035-gs-local-develop-no-docker.md) document.
 
->**NOTE:** For more details about testing, go to the [Testing Kyma](docs/kyma/docs/026-details-testing.md) document.
+>**NOTE:** For more details about testing, go to the [Testing Kyma](docs/kyma/docs/013-details-testing.md) document.
```

Comment:
link here https://kyma-project.io/docs/latest/root/kyma#details-testing-kyma
we should not promote reading from github

Reply:
Ok. In that case, I also changed links for the local and cluster installation. I guess we should talk on the Content SIG on that approach as well as I can expect no one knows that is the new approach.